packages/amplify_datastore/example/integration_test/utils/setup_utils.dart

```diff
@@ -34,6 +58,38 @@ Future<void> configureDataStore() async {
 ///
 /// see: https:///github.com/aws-amplify/amplify-android/issues/1464
 Future<void> clearDataStore() async {
-  await Future.delayed(Duration(milliseconds: 100));
+  await Future.delayed(DELAY_TO_CLEAR_DATASTORE);
   await Amplify.DataStore.clear();
 }
+
+/// Am async operator that starts DataStore API sync.
+/// It returns a Future which complets when [Amplify.Hub] receives the ready
+/// event on [HubChannel.DataStore], or completes with an timeout error if
+/// the ready event hasn't been emitted with in 2 minutes.
+class DataStoreStarter {
+  final Completer _completer = Completer();
+  late StreamSubscription hubSubscription;
+
+  Future<void> startDataStore() async {
```

Comment:
nit: can `async` be removed here as this returns future without calling `await`?